hw/top_darjeeling/lint/top_darjeeling_dv_lint_cfgs.hjson

```diff
@@ -0,0 +1,220 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+
+  // This is the primary cfg hjson for DV linting. It imports ALL individual lint
+  // cfgs of the IPs DV environments and the full chip DV environment for top_darjeeling.
+  // This enables to run them all as a regression in one shot.
+  name: top_darjeeling_dv_batch
+
+  import_cfgs:      [// common server configuration for results upload
+                     "{proj_root}/hw/data/common_project_cfg.hjson"
+                     // tool-specific configuration
+                     "{proj_root}/hw/lint/tools/dvsim/{tool}.hjson"]
+
+  flow: "lint"
+
+  // Different dashboard output path for each tool
+  rel_path: "hw/top_darjeeling/dv/lint/{tool}/summary"
+
+  use_cfgs: [{    name: dma
+                  fusesoc_core: lowrisc:ip:dma_sim
+                  import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
+                  rel_path: "hw/ip/dma/lint/{tool}"
+             },
+          //    {    name: mbx
```

Comment:
Would it make sense to open an issue about the comment characters in this list so that we can't accidentally forget to turn on these lint configurations once we've merged the blocks / turned on the top-level autogen ones?

Reply:
Done in https://github.com/lowRISC/opentitan/issues/24737